.github/workflows/rust.yml

```diff
@@ -2,9 +2,9 @@ name: Continuous integration
 
 on:
   push:
-    branches: [ master ]
+    branches: [ main ]
```

Comment:
The changes to this file may have been added by mistake.

Reply:
Oops, yeah I renamed the master branch in our fork, because we don't do master branches no more. Let me see how to fix this.